### ISSUE TITLE
run Gratipay 191

### ISSUE THREAD

**Issue by rohitpaulk** [gratipay/inside.gratipay.com#485]
[190](https://github.com/gratipay/inside.gratipay.com/issues/477)

**Comment by rohitpaulk** [comment]
Let's try this on a droplet today..

**Comment by rohitpaulk** [comment]
I see an image named 'payday', lets see what it contains

**Comment by whit537** [comment]
:airplane:

**Comment by rohitpaulk** [comment]
I tried resetting the root password, but still not able to login.

**Comment by whit537** [comment]
D'oh!

**Comment by rohitpaulk** [comment]
I guess we've disabled password-auth completely..

**Comment by rohitpaulk** [comment]
https://github.com/gratipay/inside.gratipay.com/blob/a7707049a57c168ffe283e5cc0e9da1953196d1d/www/howto/run-payday.spt#L27

> `ssh payday` (the image has a key of mine on it under the `whit537` user).

**Comment by rohitpaulk** [comment]
Lemme spin up a new droplet instead. Shouldn't be very hard to get started, let's see..

**Comment by rohitpaulk** [comment]
Okay! I have gratipay up and running on the droplet. Added SSH keys to github and heroku too.

**Comment by rohitpaulk** [comment]
Tips migrated. 

```
$ cat migrate-tips.log
Done.
```

!m @aandis

**Comment by rohitpaulk** [comment]
1.0 payouts reviewed. No ~users with subscriptions setup to be reviewed.

**Comment by rohitpaulk** [comment]
Backup triggered on heroku.

**Comment by rohitpaulk** [comment]
Script done.

```
pid-12861 thread-140336613721920 (MainThread) Script ran for 53 seconds (0:00:53.409115).
```

**Comment by whit537** [comment]
![screen shot 2016-01-28 at 11 51 16 am](https://cloud.githubusercontent.com/assets/134455/12651659/76ea31b0-c5b5-11e5-8b05-31c61a5fbc1c.png)

**Comment by rohitpaulk** [comment]
Logs uploaded in https://github.com/gratipay/logs/commit/b032a427371f31ef23fd42ca80d0fd236460504e

**Comment by rohitpaulk** [comment]
We have `NULL` fee caps for the 1.0 payouts that I just reviewed..

**Comment by rohitpaulk** [comment]
Manually updated them all to 20, I'll make sure to check them against the Masspay UI.

**Comment by rohitpaulk** [comment]
<img width="1117" alt="screen shot 2016-01-28 at 10 49 08 pm" src="https://cloud.githubusercontent.com/assets/3893573/12652594/7f207f5e-c611-11e5-85ec-0a6a08a3d188.png">

Decline rate = 7/(68 + 7) = 7.1%

**Comment by rohitpaulk** [comment]
MassPay done and posted back for 56 users.

**Comment by rohitpaulk** [comment]
1715 (today's masspay) * 5 = 8575, and we have $7.5k available in Paypal. I'm going to add $1000

**Comment by rohitpaulk** [comment]
Masspay logs uploaded in https://github.com/gratipay/logs/commit/56f6db1d9cdc9150fda2aa88f330e54475b35552

**Comment by rohitpaulk** [comment]
Droplet and SSH keys destroyed.

**Comment by whit537** [comment]
!m @rohitpaulk 

You're a pro. :-)

**Comment by rohitpaulk** [comment]
:)

**Comment by rohitpaulk** [comment]
I struggled a bit to get a working Gratipay copy on DO today, leaving notes here for reference. 

I tried Ubuntu first, but the digitalocean mirrors for `apt-get` were down, so I switched to CentOS. When running `make env`, I think I hit https://github.com/pypa/virtualenv/issues/565. I removed the [`--always-copy` option in the Makefile](https://github.com/gratipay/gratipay.com/blob/master/Makefile#L24), and setup proceeded fine. I had to install a few libraries - `postgres-server`, `postgres-devel`, `gcc`, `gcc-c++` are the ones that I remember.
